configsamples/l2adv_selects_pools_by_list.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: metallb.io/v1beta1
+kind: L2Advertisement
+metadata:
+  name: example
+  namespace: metallb-system  
+spec:
+  ipAddressPools:
+  - first-pool
```

Comment:
Is this the name of the ipaddresspool?  If so maybe give it the name of the simple ipaddresspool above "example".  And to clarify think about changing the name in the above ipaddresspool-simple "example" to something to that explicit.